cruise.umple/test/cruise/umple/compiler/UmpleParserTest.java

```diff
@@ -444,7 +444,21 @@ public void immutableClass()
     Assert.assertEquals(true,model.getUmpleClass("Student").isImmutable());
     Assert.assertEquals(false,model.getUmpleClass("Mentor").isImmutable());
   }
-
+  
+  
+  //Issue 1152
+  @Test
+  public void propagateImmutabilityToAllRelationships(){
+    assertSimpleParse("022_propagateImmutabilityToAllRelationships.ump");
```

Comment:
Where is this file?

Reply:
This file is under the test/cruise/umple/compiler. The content of it are the second example posted under the issue.